anaconda_anon_usage/utils.py

```diff
@@ -172,9 +178,12 @@ def _saved_token(fpath, what, must_exist=None):
             _debug("Retrieved %s token: %s", what, client_token)
         except Exception as exc:
             _debug("Unexpected error reading: %s\n  %s", fpath, exc, error=True)
+    if not client_token and read_only:
+        _debug("Read-only %s token does not exist", what)
+        return client_token
```

Comment:
I'm not seeing in the PR what this is used for, could you elaborate?

Reply:
Let me investigate; it might be an artifact of my previous PR